community_images/common/tests/common_commands.sh

```diff
@@ -1,32 +1,30 @@
-#!/bin/bash
+#!/bin/sh
 
 set -x
 set -e
 
 # Add common commands here which should be present in all hardened images
 
-function run_commands()
+run_commands()
 {
-    local cmd="$1"
-    if command -v "$cmd" --help &> /dev/null
+    cmd="$1"
+    if command -v "$cmd" --help > /dev/null
     then
         "$cmd" --help
     fi
 }
 
 sleep 1
 
-declare -a command_array=(
```

Comment:
Same here, why this change is required?

Reply:
When changing the bash script to shell, this is the only way we can define an array, as shell doesn't support array's natively.
we needed the shell as the alpine image used by traefik doesn't have bash